Makefile

```diff
@@ -46,7 +46,7 @@ lint_terraform: dev ## Lint the terraform files.
 	@terraform/scripts/lint.sh
 
 lint_shellcheck:
-	find . -name '*.sh' -not -path '*/vendor/*' -not -path './platform-tests/pkg/*'  -not -path './manifests/cf-deployment/*' | xargs shellcheck
+	find . -name '*.sh' -not -path '*/vendor/*' -not -path './platform-tests/pkg/*'  -not -path './manifests/cf-deployment/*' -not -path './.venv/*' | xargs shellcheck
```

Comment:
What pulls in the venv - why's it there?

Reply:
I couldn't get the python stuff to work using the global (or per-user) python environment. Using a virtualenv was the only way I could get `make test` to work locally. The problem then was: shellcheck threw literally hundreds of errors checking the contents of .venv (which contain lots of shell scripts). So I wanted to exclude those, and couldn't find a better way.

Reply:
You could instantiate the venv outside the repo, I *believe* ...

Reply:
TL;DR: I've taken the change out of this PR.

I thought about this over the weekend. Putting a venv outside of the repo is not idiomatic and certainly not something I want to encourage. So I think the best thing to do here is to add some scripting and documentation so that everyone then uses a venv in the same place, so a change like this is not so out of place. I'll do that in a separate PR when I get a moment.